### PR TITLE
Keypoints stream

### PR DESCRIPTION
**Pull request by pkarimib**

### CODE REVIEW

File: src/aiortc/codecs/keypointcodec.py
Comment (vibhaa) [inline src/aiortc/codecs/keypointcodec.py:20]: please break this line up - in fact would recommend defining intermediary variables for the int version of the bytearray variables and why the indices are what they are, so its cleaerer

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [inline src/aiortc/contrib/getkeypoints.py:21]: smaller lines please

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:161]: please add a space here

Comment (vibhaa) [reply src/aiortc/contrib/media.py:161]: extra line i mean

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:262]: please add an extra line here to separate the blocks

File: src/aiortc/mediastreams.py
Comment (vibhaa) [inline src/aiortc/mediastreams.py:177]: No extra line here please.

File: src/aiortc/mediastreams.py
Comment (vibhaa) [inline src/aiortc/mediastreams.py:174]: are these values necessary/where are they used?

Comment (vibhaa) [reply src/aiortc/mediastreams.py:174]: do you need this function or can you just defer to the base class and not reeimplement?

File: src/aiortc/codecs/keypointcodec.py
Comment (vibhaa) [inline src/aiortc/codecs/keypointcodec.py:36]: No line here - please be mindful of these

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [inline src/aiortc/contrib/getkeypoints.py:18]: smaller line

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [inline src/aiortc/contrib/getkeypoints.py:45]: No line here please. At this point, I think you should be able to look out for these yourself. Please don't wait until I need to point it out. Any code in this codebase should strictly conform to their code standards.

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [reply src/aiortc/contrib/getkeypoints.py:58]: look at other files as examples

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [reply src/aiortc/contrib/getkeypoints.py:75]: if it is only done for training, it technically shouldn't even be here right?

File: src/aiortc/contrib/media.py
Comment (pkarimib) [reply src/aiortc/contrib/media.py:164]: yeah. I will fix it, thanks for your comment

File: src/aiortc/contrib/getkeypoints.py
Comment (pkarimib) [reply src/aiortc/contrib/getkeypoints.py:58]: So we don't need a line before `if` unless it's a new code block?

Comment (vibhaa) [reply src/aiortc/contrib/getkeypoints.py:58]: Yeah before depends on the code block, but after the "if" and before the "else", atypical to have a line

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [inline src/aiortc/contrib/getkeypoints.py:24]: would simplify this commenting structure as I mentioned to not have separate "inputs" because of the style of this codebase. Would rather work it into the line comment instead

Comment (vibhaa) [reply src/aiortc/contrib/getkeypoints.py:24]: same thing with returns

Comment (vibhaa) [reply src/aiortc/contrib/getkeypoints.py:24]: Yeah so this is just following up on the old review comment I already had

**Review by vibhaa — CHANGES_REQUESTED** [review]
Tiny changes.

File: src/aiortc/mediastreams.py
Comment (pkarimib) [inline src/aiortc/mediastreams.py]: This introduced a bug in my code. To be fixed!

File: examples/videostream-cli/run_exp.sh
Comment (vibhaa) [inline examples/videostream-cli/run_exp.sh:23]: why are these lines there? its the same as the above command?

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:158]: please do not add print statements in existing blank lines, rather add it as a new line to preserve readability

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:167]: Please change all prints to logs before merging - I don't want to have to make another iteration in the future converting all prints to log statements

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:170]: same as above

Comment (vibhaa) [reply src/aiortc/contrib/media.py:170]: please also make sure that any past print statements you added as part of this PR are all changed

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:428]: log please

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:422]: log please

File: src/aiortc/mediastreams.py
Comment (vibhaa) [inline src/aiortc/mediastreams.py:155]: reads constant keypoints

Comment (vibhaa) [reply src/aiortc/mediastreams.py:155]: keypoint track not keypoints track

File: src/aiortc/contrib/getkeypoints.py
Comment (pkarimib) [reply src/aiortc/contrib/getkeypoints.py:24]: removed the unnecessary comments

File: examples/videostream-cli/cli.py
Comment (pkarimib) [reply examples/videostream-cli/cli.py:88]: sure, we'll fix it in merge

File: src/aiortc/contrib/media.py
Comment (vibhaa) [inline src/aiortc/contrib/media.py:167]: Can't be sure about exact length of the line from this view, but please try to break this and the below line up so its not too long

File: src/aiortc/contrib/getkeypoints.py
Comment (vibhaa) [inline src/aiortc/contrib/getkeypoints.py:22]: very minor suggestion: would shift the "and" to match the place where Generates starts, but this is truly a personal style.

Comment (pkarimib) [reply src/aiortc/contrib/getkeypoints.py:22]: done

**Pull request merged**
